### PR TITLE
Phone number validation

### PR DESCRIPTION
**Pull request by kpbacode**

### CODE REVIEW

File: app/controllers/kaui/accounts_controller.rb [hunk 1]
```diff
@@ -54,6 +54,17 @@ def new
   end
 
   def create
+
+    test_phone = KauiPhoneValid.new
+    test_phone.phone = params[:phone]
+
+
+    unless test_phone.valid?
+      flash.now[:error] = "Phone number is invalid"
```
Comment (pierre) [inline app/controllers/kaui/accounts_controller.rb:63]: Should this be something like `test_phone.errors.messages` to pick up the `global.errors.phone_format` from the model?

File: app/controllers/kaui/accounts_controller.rb [hunk 2]
```diff
@@ -54,6 +54,17 @@ def new
   end
 
   def create
+
+    test_phone = KauiPhoneValid.new
```
Comment (pierre) [inline app/controllers/kaui/accounts_controller.rb:58]: Shouldn't the validation be in the `Kaui::Account` model instead? That way, it's easily extensible to other fields.

**Review by pierre — COMMENTED** [review]
Could you add unit tests? e.g. verify things like `+1 ...` are still supported.

File: app/controllers/kaui/accounts_controller.rb (hunk 2 above)
Comment (kpbacode) [reply app/controllers/kaui/accounts_controller.rb:58]: Done

**Pull request closed without merging**